src/entities/tag.cpp

```diff
@@ -469,7 +482,8 @@ bool Tag::store() {
     if (this->id > 0) {
         query.prepare(
                 "UPDATE tag SET name = :name, priority = :priority, "
-                        "parent_id = :parentId, " + colorField + " = :color "
+                        "parent_id = :parentId, " + colorField + " = :color, "
+                        "created = datetime('now') "
```

Comment:
The created date currently (in the `pbek` repository) holds the date when the entity was created (set in the table create statement) and should not be misused as `updated` field. I guess the best would be you add a field per entity where you need it that really reflects what you are storing into it. Since you are setting it in the `store` method the name `updated` would in my opinion be appropriate.